temp/gptq_distributed_test_getmodel.py

```diff
@@ -0,0 +1,58 @@
+import os
```

Comment:
is this needed as part of the PR? I'd rather this be some unit test for all this new code